README.md

```diff
@@ -1,5 +1,7 @@
 # ansible-collection-icinga
 
+[![CI](https://github.com/Icinga/ansible-collection-icinga/workflows/CI/badge.svg?event=push)](https://github.com/Icinga/ansible-collection-icinga/actions?query=workflow%3ACI)
```

Comment:
The CI Status doesn't work, this generic one does work. 
https://github.com/Icinga/ansible-collection-icinga/actions/workflows/molecule.yml/badge.svg

Reply:
The other badge should work, once it's merged. The pipelines the link is referring to, does not yet exist. But we can always change it to the more generic one to be safe. I'll change it.